---
title: research
bg: blue
color: white
style: left
fa-icon: bar-chart
---

<p>
An updated version of my <strong>CV</strong> (including publications) can be found <a href="https://github.com/pimentel/cv/raw/master/academic/HaroldPimentel_cv.pdf" target="_blank">here</a>.
</p>
<p>
Generally, I am interested in computational genomics and the role that high dimensional statistics, algorithms, and machine learning play in modern biology.
</p>
<p>
Most of my graduate work has focused on general methods for RNA-Seq analysis.
I have worked on transcript level differential expression, transcript quantification, transcriptome alignment, alternative splicing estimation, intron retention estimation, and other related problems.
</p>


### __transcript level differential expression analysis__

<div class="container" align="center">
  <a href="img/differential_expression.png" target="_blank">
    <img src="img/differential_expression.png" width="300px">
  </a>
</div>

<br>

A major challenge in transcript level differential expression analysis with RNA-Seq is that we are often in a small sample setting (a total of 6 samples is very common) which results in unstable estimators and poor statistical power.
Traditionally, methods have been developed to perform shrinkage across all of the features (e.g. genes or transcripts) in order to reduce the variability in estimates and gain some power.
One thing that these methods often fail to do is to decompose the variability into biological variability and inferential variability (resulting from transcript abundance estimation).
In our method for differential expression analysis, <strong>sleuth</strong>, we are able to estimate the inferential and sampling variability using kallisto and do shrinkage directly on the biological variability.
This shrinkage estimation leads to increased power when performing transcript level differential expression analysis.

If you'd like to learn more, <a href="https://liorpachter.wordpress.com/2015/08/17/a-sleuth-for-rna-seq/" target="_blank">check out Lior's blog post</a>. A preprint is coming very soon.

### __transcript abundance estimation__

<div class="container" align="center">
  <a href="img/bootstrap.png" target="_blank">
    <img src="img/bootstrap.png" width="300px">
  </a>
</div>

<br>

<strong>Kallisto</strong> is a very fast RNA-Seq transcript abundance estimation tool that eliminates the need to map reads by using a process called pseudoalignment.
A pseudoalignment is simply an indicator of what transcripts a read is compatible with.
In addition to the pseudoalignment, the expectation-maximization (EM) algorithm is performed on a simplified likelihood based off of "equivalence classes."
This is important because this allows the EM algorithm to scale in complexity of the transcriptome (number of equivalence classes) rather than the number of reads.
In fact, this added efficiency is partially what makes sleuth possible.
Due to the reduced memory footprint of this representation, we can very rapidly resample the data to produce bootstrap estimates of the inferential variability and sampling variability that is inherent in RNA-Seq.
If you'd like to learn more, <a href="http://arxiv.org/abs/1505.02710" target="_blank">check out our preprint</a>.


### __intron retention__

<div class="container" align="center">
  <a href="img/sf3b1igv.png" target="_blank">
    <img src="img/sf3b1igv.png" width="300px">
  </a>
</div>

<br>

Over the last few years there has been some interest in identifying intron retention using RNA-Seq.
While there are several methods for quantifying these introns if they are already annotated, there aren't any for identifying novel retention events.
We developed a method, <strong>KeepMeAround</strong> to identify retained introns.
This project started by our curiosity of the classes of genes that retain introns in terminal erythropoiesis.
While the method is simply an extension of transcript abundance estimation, many of the events that were identified in RNA-Seq were validated using other protocols.
If you'd like to learn more, <a href="http://nar.oxfordjournals.org/content/early/2015/11/02/nar.gkv1168.full" target="_blank">check out our paper</a>.
